### PR TITLE
Update sklmessages_pt_BR.properties

### PR DESCRIPTION
**Pull request by dhavyzhn**

### CODE REVIEW

File: assets/launcher/lang/sklmessages_pt_BR.properties [hunk 1]
```diff
@@ -69,74 +69,74 @@ alert.button.cancel=Cancelar
 
 # About
 about.translation=Traduzido por:
-about.translation.authors=TheusKhan
+about.translation.authors=DhavyZhn
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_pt_BR.properties:72]: Why you removed TheusKhan?

**Comment by skmedix** [comment]
Im not speaking Português so I can't judge what PR is better. Im merging this one because it was first. Thank you.

@QCoisa506 @Gavuriiru You can review this, and make an additional Pull Requests with some fixes. Thanks for your effort too.

**Comment by Gavuriiru** [comment]
Hello, I'm sorry but his commit is overloaded with spelling mistakes, it's pretty hard to fix it all. It's better to just revert it and accept the content of mine and not his.
In his translation there are words that do not exist, missing accents, mistakes in agreement, gender and many other mistakes.
Thanks and sorry for the inconvenience.

**Comment by TheMatheusDev** [comment]
> @Gavuriiru Update has been reverted until I get a second opinion from TheusKhan. Thank you

I totally agree with @Gavuriiru, the commit has several errors as he mentioned.
I suggest we keep @Gavuriiru's version.

**Comment by skmedix** [comment]
@TheMatheusDev Thanks!

**Pull request merged**